pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/JUnitAssertionsShouldIncludeMessage.xml

```diff
@@ -426,4 +426,18 @@ public class TestNgTestCase {
 }
         ]]></code>
     </test-code>
+    <test-code>
+        <description><![CDATA[
+Issue #1009: JUnitAssertionsShouldIncludeMessage - False positive with assertEquals
```

Comment:
I'll change the title of the issue to reflect, that it's actually a false positive involving JUnit5. Could you please change the description here as well?
Btw. the CDATA is not needed here - without it, the description becomes a bit more readable.

Reply:
Done.

Reply:
Done.